maintainers/maintainer-list.nix

```diff
@@ -649,6 +649,12 @@
       fingerprint = "B0D7 2955 235F 6AB5 ACFA  1619 8C7F F5BB 1ADE F191";
     }];
   };
+  aimpizza = {
+    email = "rickomo.us@gmail.com";
+    name = "Rick Omonsky";
+    github = "AimPizza";
+    githubId = "64905268";
```

Comment:
```suggestion
    githubId = 64905268;
```

Reply:
What I do wonder though, is squashing and force-pushing after every change the correct approach?

Reply:
Yeah, that's the usual workflow :)